content/python/concepts/built-in-functions/terms/issubclass/issubclass.md

```diff
@@ -0,0 +1,49 @@
+## **issubclass()**
+used to check if a class is a subclass of another class or not.
+
+### Syntax
+`issubclass()` take two parameters:
+
+`issubclass(class, classinfo)`
+- **class** - class to be checked
+- **classinfo** - class, type, or tuple of classes and types 
+
+**Return type:** `True` if object is subclass of a class, or any element of the tuple, otherwise `False`.
+
+
+### Example
+
```

Comment:
```suggestion
## Example

The following example uses the `issubclass()` function to confirm whether the `myInfo` is a subclass of the `myAge` class:
```

For example sections, it's always important to lead a code example with 1-2 sentences 😄